requirements.txt

```diff
@@ -110,6 +114,7 @@ python-consul==1.1.0
 python-dateutil==2.6.0
 python-glanceclient==2.5.0
 python-keystoneclient==3.8.0
+python-magic==0.4.15
```

Comment:
@giovannicimolin, why is `python-magic` needed? It is unused in this PR and appears to be unrelated.